pkg/build/build.go

```diff
@@ -350,6 +363,73 @@ func pullImage(ctx *buildContext, image string) (id string, err error) {
 	return pushTarget.Context().Digest(digest.String()).String(), nil
 }
 
+func isImageRemote(ctx *buildContext, image string) (bool, error) {
+	ref, err := images2.ParseReferenceNoDefault(image)
+	if err != nil {
+		return false, nil
```

Comment:
I don't return the error here because this error can happen if there is no registry supplied in the image string (i.e. if it's just `myimage:latest`). This is not a problem for local images.